baseline-refaster-rules/src/test/java/com/palantir/baseline/refaster/AssertjArrayEqualsTest.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.refaster;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class AssertjArrayEqualsTest {
+    @Test
+    public void sanity_check() {
+        byte[] bytes = {1, 2, 3};
+        // both of these work, but I think isEqualTo reads a bit more nicely
```

Comment:
+1. containsExactly makes more sense if we deconstruct array creation into varargs. e.g. `assertThat(bytes).containsExactly(1, 2, 3);` but I think that's a bit extreme for automation without providing significantly better failure information.